dist/examples/ColorTriangleRotate/index.html

```diff
@@ -1,54 +0,0 @@
-<!doctype html><html lang=en><meta charset=UTF-8><meta name=viewport content="width=device-width,initial-scale=1"><title>Color Triangle with Rotation</title><body style=background-color:#000><div style=text-align:center><canvas id=cnvs width=512 height=512></canvas></div><script type=module>import { initASWebGLue, ASWebGLReady } from '../../ASWebGLue.js';
```

Comment:
Let's get some input from @battlelinegames about of these refactorings, so we can be clear to merge upstream.

Side note @battlelinegames, @ZoeDreams made an initial publish of aswebglue: https://npmjs.com/aswebglue

Reply:
Just so we're all on the same page, let's stay at `v0.x.x` until we have concensus for a `v1.x.x` release.